index.html

```diff
@@ -1,41 +1,247 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
     <title>Portfolio</title>
-    <link href="//netdna.bootstrapcdn.com/twitter-bootstrap/2.3.2/css/bootstrap-combined.no-icons.min.css" rel="stylesheet">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
-
-
-</head>
-<body class="container">
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+  </head>
+  <body class="container">
     <header class="toolbar">
-        <a class="name">Wahid Shadab</a>
-        <nav class="menu">
-            <div class="menu_item"></div>
-            <div class="menu_item"></div>
-            <div class="menu_item"></div>
-        </nav>
+      <a class="name">Wahid Shadab</a>
+      <nav class="menu">
+        <div class="menu_item"></div>
+        <div class="menu_item"></div>
+        <div class="menu_item"></div>
+      </nav>
     </header>
     <section class="headline">
-        <div class="intro">
-            <h3 class="hey">Hey there <br> I'm Shadab</h3>
-            <p class="iam">I'm a Software Developer</p>
-            <p class="about">
-                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-                don’t hestiate to contact me.
-            </p>
+      <div class="intro">
+        <h3 class="hey">
+          Hey there <br />
+          I'm Shadab
+        </h3>
+        <p class="iam">I'm a Software Developer</p>
+        <p class="about">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+      </div>
+      <ul class="social_media">
+        <li class="fa fa-linkedin fa-2x"><a href="#"></a></li>
+        <li class="fa fa-github fa-2x"><a href="#"></a></li>
+        <li class="fa fa-twitter fa-2x"><a href="#"></a></li>
+        <li class="fa fa-facebook fa-2x"><a href="#"></a></li>
+        <li class="fa fa-youtube fa-2x"><a href="#"></a></li>
```

Comment:
Optional suggestion

You can also change the font-size for the social media icons to be same as we have on the figma.